simplq/src/components/common/QueueStats/queueStats.module.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../common.scss';
+
+.detail {
+  display:flex;
+  flex-direction: column;
+  padding: 0.5rem 2rem;
+
+  .detail-row {
+    display: flex;
+    align-items: center;
```

Comment:
is this required? Let's do a `justify-content: space-between;` here

Reply:
`align-items: center;` needs to stay for now, since elements are not the same height.

`justify-content: space-between;` works well.